### ISSUE TITLE
[Bug Report] Missin elevation 0

### ISSUE THREAD

**Issue by FernandoFranco** [vuetifyjs/vuetify#9157]
### Environment
**Vuetify Version:** 1.5.18
**Vue Version:** 2.6.10
**Browsers:** Google Chrome, Mozilla Firefox, Safari, Microsoft Edge, Internet Explorer, Opera
**OS:** Linux, Mac OSX, iOS, Android, Windows

### Steps to reproduce
Create a card with elevation 0

### Expected Behavior
Card with elevation 0 (flat)

### Actual Behavior
Card with default elevation

### Reproduction Link
<a href="https://codepen.io/fernando_franco/pen/RwbzZxP" target="_blank">https://codepen.io/fernando_franco/pen/RwbzZxP</a>

### Other comments
Change line 15 in https://github.com/vuetifyjs/vuetify/blob/v1.5.18/packages/vuetify/src/mixins/elevatable.ts to:
```javascript
if (!this.computedElevation && this.computedElevation !== 0) return {}
```

**Comment by hugoazevedosoares** [comment]
on it

**Comment by hugoazevedosoares** [comment]
ps: this bug does not appear to happen on 2.x.x versions

**Comment by FernandoFranco** [comment]
> ps: this bug does not appear to happen on 2.x.x versions

**Comment by FernandoFranco** [comment]
> ps: this bug does not appear to happen on 2.x.x versions

I didn't really check in versions 2.x.x

**Comment by johnleider** [comment]
fixed in a7df4cc
